d1lod/d1lod/jobs.py

```diff
@@ -213,11 +213,13 @@ def calculate_stats():
     JOB_NAME = "JOB_GRAPH_STATS"
     logging.info("[%s] Job started.", JOB_NAME)
 
-    s = Store(SESAME_HOST, SESAME_PORT)
-    r = Repository(s, SESAME_REPOSITORY)
+    s = Store(VIRTUOSO_HOST, VIRTUOSO_PORT, VIRTUOSO_REPOSITORY, ns=NAMESPACES)
+    r = Repository(host=VIRTUOSO_HOST, port=VIRTUOSO_PORT, name=VIRTUOSO_REPOSITORY, ns=NAMESPACES)
     Interface(r)  # Adds namespaces we need to repo
 
-    logging.info("[%s] repository.size=%d", JOB_NAME, r.size())
+    # have no feature to determine the repository size
```

Comment:
Could this either get implemented and put back in or just removed since we're using version control and don't really needed commented-out code.

Reply:
Resolved by implementing the `size()` function